apiary.apib

```diff
@@ -5943,6 +5939,77 @@ For a more detailed guide on the file upload, see the corresponding part of the
                 }
             }
 
++ Response 201 (application/json)
```

Comment:
Sample odpovědí vytvoření file resource pro Azure a GCS. Nepřišel jsem ale na to jak je nějak pojmenovat.

Reply:
Hmm toto je takové blbé ty multiple response nevím jestli to někde používáme apiary to hlásí jako warn, co na to @tomasfejfar

Reply:
Zkoušel jsem to v editoru a vykreslí se to ok, warningu jsem si nevšiml. (Nicméně beru Apiary jako legacy kterýho se stejně chceme zbavit, tak nějaký warning možná neva. 🙂) Ale pokud byste to raději jinak (např. rozepsat na samostatný requesty), tak to klidně udělám, to není problém. Jen jsem to nechtěl dělat dopředu, abych to pak nepřepisoval. Tak si spíš řekněte jak.

Reply:
No on je ten apiary dost debilní, mě to taky nijak nevadí ten warning, ale tuším sme měli nějak domluvené že warningy přidávat nebude, každopádně druhý způsob jak to udělat, dat tam ten endpoint víckrát je taky dost na prd.
nevíš @romanbracinik?

Reply:
OK, já stým souhlasím a pustil bych to teda dál.